actions/image_partition_action.go

```diff
@@ -342,8 +342,17 @@ func (i ImagePartitionAction) formatPartition(p *Partition, context debos.DebosC
 
 	cmdline := []string{}
 	switch p.FS {
+	case "fat16":
+	case "fat32":
```

Comment:
probably makes sense to add fat12 as well for completeness and maybe fat which relies on the mkfs autoselection?

Reply:
yeah adding `fat` here changes behaviour so I decided to keep this commit minimal. I will add it.